packages/peregrine/lib/talons/CreateAccount/useCreateAccount.js

```diff
@@ -62,20 +59,18 @@ export const useCreateAccount = props => {
                 });
 
                 // Sign in and save the token
-                const response = await signIn({
+                const signInResponse = await signIn({
                     variables: {
                         email: formValues.customer.email,
                         password: formValues.password
                     }
                 });
 
                 const token =
-                    response && response.data.generateCustomerToken.token;
+                    signInResponse &&
+                    signInResponse.data.generateCustomerToken.token;
```

Comment:
Potential cause of `TypeError`? Also `generateCustomerToken` sounds like a `function` not an `object`. If possible can we change the name?

Reply:
Seems outside of the scope of this PR, but I can investigate. I was expecting `signIn` to `reject` on `error` and enter the `catch`, but its possible it may still `resolve` and expect us to destructure `error` and handle ourselves.

Nothing we can do about that name; data comes back with the name of the mutation, which is `generateCustomerToken`. I'll sanity check that assumption though.

Reply:
> data comes back with the name of the mutation

You might be able to modify that through aliasing:

```
query storeConfigData {
  foo: storeConfig {
    id
    copyright
  }
}
```
 returns
```
{
  "data": {
    "foo": {
      "id": 1,
      "copyright": "Copyright © 2013-present Magento, Inc. All rights reserved."
    }
  }
}
```

Reply:
You could do this and alias as `customerToken: generateCustomerToken (...)` but I wonder if maybe it's a red flag that we're even asking this question. The GQL docs look like leaving the verb form query name is fine: https://graphql.org/learn/queries/#mutations

It makes it easier to grok when you consider that we may be composing several queries or mutations at once. We may need some layer that is able to understand what queries have returned with data without understanding what data they return.